great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -541,7 +541,7 @@ def plugins_directory(self) -> Optional[str]:
         )
 
     @property
-    def stores(self) -> dict:
+    def stores(self) -> dict[str, dict]:
```

Comment:
Although correct (I think) this reveals too many typing errors.
Will revert this.